### PR TITLE
Add the exclude acct in recursive function

### PR DESCRIPTION
**Pull request by DanSallau**
Hi,

Please attached contained few changes that should address our exluded acct issue for your perusal.

### CODE REVIEW

File: src/_store/root.js [hunk 1]
```diff
@@ -24,6 +24,25 @@ addLocaleData({
 
 const emptyObject = {};
 
+
+export const accountExclusion = async (token, accts = []) => {
```
Comment (borisyankov) [inline src/_store/root.js]: Does that need to be in root.js ?

Comment (nuruddeensalihu) [inline src/_store/root.js]: Nope, we can take it to utils.

Comment (nuruddeensalihu) [inline src/_store/root.js]: @borisyankov i think we cannot take it binary-utils. Also i don't think leaving there would do harm, or should i take it to a diff file?

Comment (qingweibinary) [inline src/_store/root.js]: Can we improve the parameter name?

does token mean token to try next or?
what does accts means and how is it different from window.BinaryBoot.accounts

File: src/settings/SettingsSelfExclusion.js [hunk 2]
```diff
@@ -74,7 +74,12 @@ export default class SettingsSelfExclusion extends PureComponent {
 				success: true,
 				serverError: false,
 			});
-			setTimeout(() => this.setState({ success: false }), 3000);
+			setTimeout(() => {
```
Comment (qingweibinary) [inline src/settings/SettingsSelfExclusion.js]: I never notice this before, why do we need a settimeout here?

Comment (borisyankov) [inline src/settings/SettingsSelfExclusion.js]: Not suru. Maybe @nuruddeensalihu added it initially too?

Comment (nuruddeensalihu) [inline src/settings/SettingsSelfExclusion.js]: @borisyankov i didn't add . The setTimeout there is what you use for splash message in self exclusion. It was added by you . 😉 . Btw have refactored the var names.

**Comment by qingweibinary** [comment]
@borisyankov @nuruddeensalihu 

anymore issue with this PR?

If not we merge it?

I still have question of the 3 seconds timeout though, nuru, do you think we need it? if not how about we remove it.

**Comment by borisyankov** [comment]
Yeah, my only comment now is the setTimeout.

**Comment by nuruddeensalihu** [comment]
@borisyankov @qingweibinary  the setTimeOut is there in our code at the moment. You can verify that. Like i said when you update selfExclusion, it animate the message for a while which disappear after few seconds. I believe the purpose it served there . 😉

**Comment by qingweibinary** [comment]
@nuruddeensalihu I see, that's fine then, thanks

**Pull request merged**